src/wxvx/workflow.py

```diff
@@ -160,18 +160,26 @@ def _grib_index_file(outdir: Path, url: str):
 @task
 def _grid_grib(c: Config, tc: TimeCoords, var: Var):
     yyyymmdd, hh, leadtime = tcinfo(tc)
-    outdir = c.paths.grids_baseline / yyyymmdd / hh / leadtime
-    path = outdir / f"{var}.grib2"
-    taskname = "Baseline grid %s" % path
+    url = c.baseline.url.format(yyyymmdd=yyyymmdd, hh=hh, fh=int(leadtime))
+    scheme, src = _classify_url(url)
```

Comment:
```suggestion
    proximity, src = _classify_url(url)
```
`scheme` would be `http://`, `file://`, etc.